### ISSUE TITLE
On month or year change event

### ISSUE THREAD

**Issue by hamczu** [onefinestay/react-daterange-picker#111]
Is there an event to listen to changes made by user in range picker's UI?
For example I would like to know when user changes month - I want to render some additional info related with that month. 

And I'm not talking about selection - this happens before user selects anything.
I checked the source code and I don't see that event but maybe I'm missing something.

**Comment by AlanFoster** [comment]
@hamczu If we're missing any events that you think others could benefit from, we're more than happy to accept PRs which would add the appropriate hooks you need

**Comment by hamczu** [comment]
Yep, I was prepared to write that code - I just didn't want to reinvent the wheel so that's why I asked first.
Thank you for the fast answer, now I know that I need to implement it by myself.
